charmcraft.yaml

```diff
@@ -1,13 +1,24 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file configures Charmcraft.
-# See https://juju.is/docs/sdk/charmcraft-config for guidance.
-
 type: charm
 bases:
   - build-on:
-    - name: ubuntu
+    - name: "ubuntu"
       channel: "22.04"
```

Comment:
Can we target 24.04? Same for `run-on`.

Reply:
updated to 24.04, the catch is I'd have to rewrite `charmcraft.yaml` and use charmcraft 3.0.0.